.env.sample

```diff
@@ -0,0 +1,2 @@
+BINTRAY_USER=
+BINTRAY_KEY=
```

Comment:
We should add a `README` with instructions about those. I mean in practice those will likely be provided by CI rather than run locally, but still.